page.tsx

```diff
@@ -19,6 +20,10 @@ export default function createPage(swingsetOptions: SwingsetOptions = {}) {
     const baseRoute = useBaseRoute()
     const [filterValue, setFilterValue] = useState<string | undefined>()
     const searchInputRef = useRef<HTMLInputElement>(null)
+    const [isFullscreen, setIsFullscreen] = useState(false)
+    const handleFullscreenBttnClick = () => {
+      setIsFullscreen(!isFullscreen);
+    }
```

Comment:
💭 What if we used a URL param to toggle sidebar visibility? That would allow us to share a fullscreen version of a component with design with ease.

Reply:
Brilliant - I'll add that.

Reply:
Done and done!

Reply:
Can we toggle the `isFullscreen` url param to the URL on change? Looks like we may only be reading it?